packages/genesys-spark-components/src/components/stable/gux-pagination-cursor/gux-pagination-cursor.tsx

```diff
@@ -64,18 +74,23 @@ export class GuxPaginationCursor {
   async componentWillLoad(): Promise<void> {
     trackComponent(this.root);
 
+    if (!this.controls) {
+      logWarn(this.root, 'A controls prop may help with accessibility');
+    }
+
     this.i18n = await buildI18nForComponent(this.root, translationResources);
   }
 
   renderSimpleLayout(): JSX.Element {
     return [
-      <div role="navigation" class="gux-pagination-button-container">
+      <nav aria-label={this.label} class="gux-pagination-button-container">
         <gux-button-slot accent="ghost">
           <button
             class="gux-simple-button"
             type="button"
             disabled={!this.hasPrevious}
             onClick={() => this.onButtonClick('previous')}
+            aria-controls={this.controls}
```

Comment:
Would you have to check if the elements/string passed to the `controls` prop exists in the DOM  as it may be pointing to a non-existent element potentially ?

Reply:
I am not sure has something similar been done in `gux-tooltip` related to this but might be worth checking it out.